client/my-sites/site-settings/settings-security/main.jsx

```diff
@@ -63,20 +64,34 @@ const SiteSettingsSecurity = ( { rewindActive, site, siteId, siteIsJetpack, tran
 
 	return (
 		<Main className="settings-security__main site-settings">
-			<QueryRewindStatus siteId={ siteId } />
+			<QueryRewindState siteId={ siteId } />
 			<DocumentHead title={ translate( 'Site Settings' ) } />
 			<JetpackDevModeNotice />
 			<SidebarNavigation />
 			<SiteSettingsNavigation site={ site } section="security" />
-			{ rewindActive && <Backups /> }
+			{ ( includes( [ 'active', 'awaitingCredentials', 'provisioning' ], rewindState.state ) ||
+				( 'unavailable' === rewindState.state &&
+					'vp_active_on_site' === rewindState.failureReason ) ) && (
+				<Backups
+					rewindStatus={
+						'unavailable' === rewindState.state ? rewindState.reason : rewindState.state
+					}
```

Comment:
It's probably best to let the credentials form query the rewind state on its own instead of passing it down through props here so we can eliminate confusing data flows.